portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jie Liang!</h1>");
+    
+    comments = new ArrayList<>();
```

Comment:
These comments seem a constant. Just as a discussion, are there other places that we can put these at?

Reply:
I could initialize with the list of comments?

Reply:
I see that you've moved it as class attribute.